driver/InputFlags.cpp

```diff
@@ -292,6 +292,165 @@ TensorParameters InputFlags::GetValueTensor(const std::string& long_name) const
 
     MIOPEN_THROW("Too many tensor descriptor parameters.");
 }
+
+TensorParametersUint64 InputFlags::GetValueTensorUint64(const std::string& long_name) const
+{
+    const auto& input     = MapInputs.at(FindShortName(long_name));
+    const auto components = miopen::SplitDelim(input.value.c_str(), ',');
+
+    if(components.size() < 1)
+        return {};
+
+    auto parse = [](auto line) {
+        auto ret        = std::vector<uint64_t>{};
+        const auto strs = miopen::SplitDelim(line, 'x');
```

Comment:
why is the delimiter not `,` in this case?

Reply:
The GetValueTensor is a function that receives the size of the tensor, the size and layout of the tensor, and the size and stride of the tensor, such as (2x3x4) or (2x3x4, NCW) or 2x3x4, 12x4x1. Therefore, we use x and , to distinguish them.